about.md

```diff
@@ -1,30 +1,101 @@
 ---
 layout: page
-title: About the GeoBlacklight Community
+title: About GeoBlacklight
 permalink: /about/
 ---
 
-- [Community Structure](#community-structure)
-- [Community Values and Technical Values](#community-values-and-technical-values)
-- [Project Roadmap](#project-roadmap)
+GeoBlacklight is a world-class discovery platform for geospatial content, including GIS data and maps. It is an open, multi-institutional collaborative project aiming to build off of the successes of the [Blacklight](https://projectblacklight.org/) discovery interface and the [OpenGeoMetadata](https://github.com/OpenGeoMetadata) federated metadata sharing communities.
 
-GeoBlacklight is a multi-institutional, open source geospatial data discovery software application. Based on the open source software project Blacklight, GeoBlacklight began in 2014 and has reached its third major release in 2020. GeoBlacklight has been adopted as an interface for geospatial data discovery by over 25 academic libraries and cultural heritage institutions.
+GeoBlacklight began in 2014 as a collaboration by MIT, Princeton, and Stanford. As of 2020, over 25 academic libraries and cultural heritage institutions have adopted GeoBlacklight.
 
-### Community Structure
+- [Our Community](#our-community)
+- [Our Development Practices](#our-development-practices)
+- [Application Features](#application-features)
+- [GeoBlacklight Technical Values](#geoblacklight-technical-values)
 
-**Community Coordinator: [Karen Majewicz (majew030@umn.edu)](mailto:majew030@umn.edu)**
 
-The GeoBlacklight Community Coordinator holds the following roles:
+Our Community
+-------------
 
-* Provides oversight for community processes
-* Facilitates community meetings and sets agendas
-* Proposes priorities for biannual community software development sprint cycles
-* Stages project boards for community development sprints and coordinates stand-up meetings
+The GeoBlacklight Community is open to anyone interested in spatial infrastructures, libraries, GIS, maps, open source software, and related topics. Participants can contribute to the community in various ways, such as by attending meetings, writing documentation, developing metadata best practices, engaging in outreach, and writing code. Visit our [Connect page](https://geoblacklight.org/connect.html) to join our online community.
 
-### Community Values and Technical Values
+#### Events
 
-Members of the GeoBlacklight community are currently working on community values and technical values. These will be published soon.
+- **Every month:** [Zoom meetings](https://z.umn.edu/gbl-meetings) to share project updates and to discuss topical issues.
 
-### Project Roadmap
+- **2x per year:** [Community Sprints](https://github.com/geoblacklight/geoblacklight/projects?query=). These are similar to a traditional code sprint but also incorporates activities around documentation, metadata, governance, and more.
```

Comment:
Should be "These are similar to a traditional code spring but also incorporate..."

Reply:
Thanks! I removed the 's'